src/CoreLibrary/Resources/Resources.resx

```diff
@@ -252,11 +252,11 @@
   <data name="ReadyToCommunicate" xml:space="preserve">
     <value>Factory Orchestrator Service is ready to communicate with client(s).</value>
   </data>
-  <data name="RedirectingToRunAsExplorerUser" xml:space="preserve">
-    <value>{0} is a Win32 GUI program. Redirecting to run via RunAsExplorerUser...</value>
+  <data name="RedirectingToRunAsRDUser" xml:space="preserve">

```

Comment:
does this have to be 'RD' user? i see the exes is called that, but is there a reason we don't just stick to a more abstract 'remote' user and then we can avoid having to change it 'by tech'.

Reply:
good idea, ill keep the OS exe named as is, but update here :)